locales/de.json

```diff
@@ -1800,7 +1800,7 @@
 				"thankYouForYourFeedback": "Danke für dein Feedback, es wird direkt an das Entwicklungsteam der {{shortTitle}} gesendet. Bitte achte darauf, keine Passwörter und andere sensible Daten mitzusenden.",
 				"whatKindOfFeedbackDoYouWant": "Welche Art von Feedback möchtest du uns senden?",
 				"whoDoYouWantToContact": "Wen möchtest du kontaktieren?",
-				"whichAreaOfSchulCloud": "Welchen Bereich der Schul-Cloud betrifft dein Thema?",
+				"whichAreaOfSchulCloud": "Welchen Bereich der HPI Schul-Cloud betrifft dein Thema?",
```

Comment:
Isn't it HPI-Schul-Cloud?

Reply:
no it isn't

Reply:
![image](https://user-images.githubusercontent.com/43565075/109615856-ba5b6600-7b34-11eb-8cfc-91574713e74f.png)